lightning_pose/data/utils.py

```diff
@@ -25,8 +25,27 @@ class BaseLabeledExampleDict(TypedDict):
 
 class HeatmapLabeledExampleDict(BaseLabeledExampleDict):
     """Return type when calling __getitem__() on HeatmapTrackingDataset."""
+    heatmaps: TensorType["num_keypoints", "heatmap_height", "heatmap_width", float]
+
+
+class MultiviewLabeledExampleDict(TypedDict):
+    """Return type when calling __getitem__() on BaseTrackingDataset."""
+    images: Union[
+        TensorType["RGB":3, "image_height", "image_width", float],
+        TensorType["frames", "RGB":3, "image_height", "image_width", float],
+    ]
+    keypoints: TensorType["num_targets", float]
+    idxs: int
+
+
+class MultiviewHeatmapLabeledExampleDict(MultiviewLabeledExampleDict):
+    """Return type when calling __getitem__() on MultiviewHeatmapDataset."""
 
     heatmaps: TensorType["num_keypoints", "heatmap_height", "heatmap_width", float]
+    # bbx: TensorType["bounding_box", float]
+    num_views: int
+    concat_order: List[int]
+    view_names: List[int]
 
 
```

Comment:
need to add `MultiviewLabeledBatchDict` and `MultiviewHeatmapLabeledBatchDict` - note these are BATCH dicts instead of EXAMPLE dicts